zerver/lib/upload.py

```diff
@@ -355,8 +357,8 @@ def get_local_file_path(path_id):
         return None
 
 class LocalUploadBackend(ZulipUploadBackend):
-    def upload_message_image(self, uploaded_file_name, content_type, file_data, user_profile, target_realm=None):
-        # type: (Text, Optional[Text], binary_type, UserProfile, Optional[Realm]) -> Text
+    def upload_message_image(self, uploaded_file_name, uploaded_file_size, content_type, file_data, user_profile, target_realm=None):
```

Comment:
It's probably time to line-wrap the arguments for `upload_message_image`, both here and above